gps/source_manager.go

```diff
@@ -216,7 +218,7 @@ func (sm *SourceMgr) HandleSignals(sigch chan os.Signal) {
 
 				opc := sm.suprvsr.count()
 				if opc > 0 {
-					fmt.Printf("Signal received: waiting for %v ops to complete...\n", opc)
+					sm.logf("Signal received: waiting for %v ops to complete...\n", opc)
```

Comment:
No, there's no chance of it bunking up tests; the only possible way for it to occur is if a signal is sent. It's a very specific case, which is why it exists as it does in the first place.

The proper solution here involves moving direct signal handling out of gps, and into the tool-level code - not changing gps' interface for this sole logging statement.